### PR TITLE
replaced generic workload with server workload in templates

### PR DESCRIPTION
**Pull request by ksimon1**
This PR replaces generic type of workload with server workload. 
Based on this PR: https://github.com/kubevirt/common-templates/pull/55
@MarSik please review

### CODE REVIEW

File: templates/common-templates.adoc [hunk 1]
```diff
@@ -98,13 +98,13 @@ https://docs.openshift.com/enterprise/3.0/dev_guide/templates.html#creating-from
 the templates produce VM objects to be used in the cluster. The VMs
 produced from templates will have a `vm.cnv.io/template` label, whose
 value will be the name of the parent template, for example
-`fedora-generic-medium`:
+`fedora-server-medium`:
```
Comment (MarSik) [inline templates/common-templates.adoc]: Didn't we change fedora-generic to fedora-desktop?

File: templates/using-templates.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@ These templates provide a combination of some key factors that could be further
 
 The key factors which define a template are
  - Workload
-  Most Virtual Machine should be *generic* to have maximum flexibility; the *highperformance* workload trades some of this flexibility to
+  Most Virtual Machine should be *server* to have maximum flexibility; the *highperformance* workload trades some of this flexibility to
```
Comment (MarSik) [inline templates/using-templates.adoc]: server or desktop

File: templates/common-templates.adoc (hunk 1 above)
Comment (ksimon1) [reply templates/common-templates.adoc]: Fedora has all three workloads, so it depends what we choose.

Comment (MarSik) [reply templates/common-templates.adoc]: Ack, please use fedora-desktop here to have some diversity

**Comment by MarSik** [comment]
@fabiand could you please review this?

**Comment by ksimon1** [comment]
@fabiand please can you review this?

**Comment by fabiand** [comment]
Happy to do so, lgtm, thanks

**Pull request merged**